rotkehlchen/tests/unit/test_cost_basis.py

```diff
@@ -1047,3 +1060,80 @@ def test_taxable_acquisition(accountant):
+
+
+@pytest.mark.parametrize('force_no_price_found_for', [[
+    (A_ETH, Timestamp(1677593073)),
+    (A_ETH, Timestamp(1677593074)),
+    (A_ETH, Timestamp(1677593075)),
+    (A_ETH, Timestamp(1677593076)),
+]])
+@pytest.mark.parametrize('db_settings,expected_pnls', [
+    [{'cost_basis_method': CostBasisMethod.FIFO}, [ZERO, ZERO, FVal(-10), FVal(3500), ZERO, FVal(-10), ZERO, ZERO, FVal(-10), FVal(1600), ZERO, FVal(-10)]],  # noqa: E501
+    [{'cost_basis_method': CostBasisMethod.ACB}, [ZERO, ZERO, FVal(3495), ZERO, ZERO, ZERO, FVal(-6), ZERO]],  # noqa: E501
+])
+def test_fees(accountant: 'Accountant', expected_pnls: list[FVal]):
+    """
+    Tests that fees are properly either calculated as standalone events or included in the price.
+    Values for the example are taken from https://github.com/rotki/rotki/issues/5561#issuecomment-1423338938.
+
+    Note: pnls are $10 higher than in the issue since examples in the issue say that transaction
+    costs should be subtracted when selling, but in fact transaction costs are always about
+    buy cost basis https://discord.com/channels/657906918408585217/683245846070165504/1077994503849386054.
+    """  # noqa: E501
+    history: Sequence[AccountingEventMixin] = [
+        Trade(
+            timestamp=Timestamp(1677593073),
+            location=Location.EXTERNAL,
+            base_asset=A_ETH,
+            quote_asset=A_EUR,
+            trade_type=TradeType.BUY,
+            amount=AssetAmount(FVal(100)),
+            rate=Price(FVal(50)),
+            fee=Fee(FVal(10)),
+            fee_currency=A_EUR,
+            notes='Trade 1',
+        ), Trade(
+            timestamp=Timestamp(1677593074),
+            location=Location.EXTERNAL,
+            base_asset=A_ETH,
+            quote_asset=A_EUR,
+            trade_type=TradeType.SELL,
+            amount=AssetAmount(FVal(50)),
+            rate=Price(FVal(120)),
+            fee=Fee(FVal(10)),
+            fee_currency=A_EUR,
+            notes='Trade 2',
+        ), Trade(
+            timestamp=Timestamp(1677593075),
+            location=Location.EXTERNAL,
+            base_asset=A_ETH,
+            quote_asset=A_EUR,
+            trade_type=TradeType.BUY,
+            amount=AssetAmount(FVal(50)),
+            rate=Price(FVal(130)),
+            fee=Fee(FVal(10)),
+            fee_currency=A_EUR,
+            notes='Trade 3',
+        ), Trade(
+            timestamp=Timestamp(1677593076),
+            location=Location.EXTERNAL,
+            base_asset=A_ETH,
+            quote_asset=A_EUR,
+            trade_type=TradeType.SELL,
+            amount=AssetAmount(FVal(40)),
+            rate=Price(FVal(90)),
+            fee=Fee(FVal(10)),
+            fee_currency=A_EUR,
+            notes='Trade 4',
+        ),
+    ]
+    accounting_history_process(
+        accountant=accountant,
+        start_ts=Timestamp(0),
+        end_ts=Timestamp(1677593077),
+        history_list=list(history),  # converting to list to satisfy mypy
```

Comment:
what? Why do you even need to declare the sequence type in a test in the first place? Don't add unecessary stuff in the tests